pkg/controller/dataimportcron-controller.go

```diff
@@ -195,11 +198,9 @@ func getCronRegistrySource(cron *cdiv1.DataImportCron) (*cdiv1.DataVolumeSourceR
 	return source.Registry, nil
 }
 
-// FIXME: update all conditions
 func (r *DataImportCronReconciler) update(ctx context.Context, dataImportCron *cdiv1.DataImportCron) (reconcile.Result, error) {
 	log := r.log.WithName("update")
 	res := reconcile.Result{}
-	//FIXME: add ImportStatus DataVolumePhase so we'll handle only on transition (to Succeeded etc.)
 	importSucceeded := false
 	imports := dataImportCron.Status.CurrentImports
 	if imports != nil {
```

Comment:
I think conditions should be added/updated all the time, not just when `imports != nil`

Reply:
The `DataImportCronUpToDate` condition is already updated all the time. When `imports == nil` the missing condition update is `DataImportCronProgressing` = false. Adding it. Thanks.